src/lib/journey/callbacks/password/confirm-input.svelte

```diff
@@ -41,9 +43,18 @@
   }
 
   $: {
+    isInvalid =
+      passwordsDoNotMatch || (isRequired && typeof value === 'string' && value.length === 0);
     if (resetValue) {
-     value = undefined;
-     onChange(value);
+      value = undefined;
+      onChange(value);
+    }
+    if (passwordsDoNotMatch) {
+      message = interpolate('passwordConfirmationError', null, 'Passwords do not match');
+    } else if (isRequired && !value) {
```

Comment:
I'd like for you to also try removing the second portion of this condition as well. The `&& !value` is likely not needed as the core input component will not show the message unless the input is explicitly marked as invalid. I'd recommend trying to ensure the underlying components work as expected.

Reply:
This is correct, it can be removed.